app/views/articles/index.json.jbuilder

```diff
@@ -1,7 +1,9 @@
 docs = @presenter.documents.collect do |document|
   link = ArticleFulltextLinkPresenter.new(document: document, context: self).links.try(:first) # top priority one only
+  composed_title = document['eds_composed_title']
   data = document.to_h # avoids deprecation warning
   data['fulltext_link_html'] = link if link.present?
+  data['eds_composed_title'] = italicize_composed_title({ value: Array.wrap(composed_title) }) if composed_title.present?
```

Comment:
I was going to look into moving this over into the new JsonResults presenter but realized that the code change to get this updated was going to be a lot smaller than refactoring into that (or a similar) class.